lib/Db/MessageSnoozeMapper.php

```diff
@@ -57,4 +58,19 @@ public function deleteByMessageId(string $messageId): void {
 
 		$delete->executeStatement();
 	}
+
+	/**
+	 * Delete all orphaned entries that should have been unsnoozed a month ago.
+	 * We assume that these messages no longer exist in the snoozed mailbox.
+	 */
+	public function deleteOrphans(): void {
+		$deleteQb = $this->db->getQueryBuilder();
+		$deleteQb->delete($this->getTableName())
+			->where($deleteQb->expr()->lt(
+				'snoozed_until',
+				$deleteQb->createNamedParameter(($this->time->getTime() - 2592000), IQueryBuilder::PARAM_INT),
```

Comment:
```suggestion
				$deleteQb->createNamedParameter(($this->time->getTime() - (30*7*24*3600)), IQueryBuilder::PARAM_INT),
```

for readability and the de-optimization won't hurt

Reply:
:white_check_mark: Removed the 7*, otherwise we would wait 7 months ;)